src/container/srv_target.c

```diff
@@ -973,6 +973,23 @@ cont_child_destroy_one(void *vin)
 	return rc;
 }
 
+int
+ds_cont_tgt_destroy(uuid_t pool_uuid, uuid_t cont_uuid)
+{
+	struct cont_tgt_destroy_in *in;
+	int rc;
+
+	D_ALLOC_PTR(in);
```

Comment:
Passed in pool_uuid & cont_uuid are not used? How can this work?

Reply:
It doesn't work yet :) All the pieces are in place though, so I wanted to get a review of the approach before spending a bunch of time fixing bugs in code that needed to be fundamentally reworked. 

Fixed, thanks.

Reply:
why do we even allocate @in, it can be in stack right?

Reply:
Fixed

Reply:
Fixed